{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "seleniumpagefactory.Pagefactory.ElementNotFoundException: An exception of type TimeoutException occurred. With Element -: username_logged_in - locator: (xpath, //span[@data-qa='lufexuloga'])", "trace": "self = <tests.pageObjects.pageFactory.dashboardPage_pf.DashboardPage object at 0x000001752163F410>, loc = 'username_logged_in'\n\n    def __getattr__(self, loc):\n    \n        if self.mobile_test == True:\n            if loc in self.locators.keys():\n                element = self.find_element_by_accessibility_id(self.locators[loc][1])\n                return element\n        else:\n            if loc in self.locators.keys():\n                locator = (self.TYPE_OF_LOCATORS[self.locators[loc][0].lower()], self.locators[loc][1])\n                try:\n>                   element = WebDriverWait(self.driver, self.timeout).until(\n                        EC.presence_of_element_located(locator)\n                    )\n\n.venv\\Lib\\site-packages\\seleniumpagefactory\\Pagefactory.py:58: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"bd9dfbc22e172c014519d07d94fdd9e1\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x0000017521663CE0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF7F1629412+29090]\nE       \t(No symbol) [0x00007FF7F159E239]\nE       \t(No symbol) [0x00007FF7F145B1DA]\nE       \t(No symbol) [0x00007FF7F14AEFE7]\nE       \t(No symbol) [0x00007FF7F14AF23C]\nE       \t(No symbol) [0x00007FF7F14F97C7]\nE       \t(No symbol) [0x00007FF7F14D672F]\nE       \t(No symbol) [0x00007FF7F14F65A2]\nE       \t(No symbol) [0x00007FF7F14D6493]\nE       \t(No symbol) [0x00007FF7F14A09D1]\nE       \t(No symbol) [0x00007FF7F14A1B31]\nE       \tGetHandleVerifier [0x00007FF7F194871D+3302573]\nE       \tGetHandleVerifier [0x00007FF7F1994243+3612627]\nE       \tGetHandleVerifier [0x00007FF7F198A417+3572135]\nE       \tGetHandleVerifier [0x00007FF7F16E5EB6+801862]\nE       \t(No symbol) [0x00007FF7F15A945F]\nE       \t(No symbol) [0x00007FF7F15A4FB4]\nE       \t(No symbol) [0x00007FF7F15A5140]\nE       \t(No symbol) [0x00007FF7F159461F]\nE       \tBaseThreadInitThunk [0x00007FFC337A257D+29]\nE       \tRtlUserThreadStart [0x00007FFC348CAF28+40]\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test.vwoLoginTests.test_vwo_login_pf.TestVWOLogin object at 0x0000017521691100>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bd9dfbc22e172c014519d07d94fdd9e1\")>\n\n    @pytest.mark.usefixtures(\"setup\")\n    @pytest.mark.qa\n    def test_vwo_login_positive(self, setup):\n        LOGGER = logging.getLogger(__name__)\n        LOGGER.info(\"Starting the Testcase\")\n        driver = setup\n        driver.get(Constants.app_url())\n        login_page = LoginPage(driver)\n        login_page.login_to_vwo(username=self.username, pwd=self.password)\n        dashboard_page = DashboardPage(driver)\n>       username = dashboard_page.user_logged_in_text()\n\ntests\\test\\vwoLoginTests\\test_vwo_login_pf.py:46: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests\\pageObjects\\pageFactory\\dashboardPage_pf.py:19: in user_logged_in_text\n    return self.username_logged_in.get_text()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.pageObjects.pageFactory.dashboardPage_pf.DashboardPage object at 0x000001752163F410>, loc = 'username_logged_in'\n\n    def __getattr__(self, loc):\n    \n        if self.mobile_test == True:\n            if loc in self.locators.keys():\n                element = self.find_element_by_accessibility_id(self.locators[loc][1])\n                return element\n        else:\n            if loc in self.locators.keys():\n                locator = (self.TYPE_OF_LOCATORS[self.locators[loc][0].lower()], self.locators[loc][1])\n                try:\n                    element = WebDriverWait(self.driver, self.timeout).until(\n                        EC.presence_of_element_located(locator)\n                    )\n                except (StaleElementReferenceException, NoSuchElementException, TimeoutException) as e:\n>                   raise ElementNotFoundException(\n                        \"An exception of type \" + type(e).__name__ +\n                        \" occurred. With Element -: \" + loc +\n                        \" - locator: (\" + locator[0] + \", \" + locator[1] + \")\"\n                    )\nE                   seleniumpagefactory.Pagefactory.ElementNotFoundException: An exception of type TimeoutException occurred. With Element -: username_logged_in - locator: (xpath, //span[@data-qa='lufexuloga'])\n\n.venv\\Lib\\site-packages\\seleniumpagefactory\\Pagefactory.py:62: ElementNotFoundException"}, "start": 1726209888285, "stop": 1726209919506, "uuid": "7a01f237-224a-4e39-a86b-44cfc39abee1", "historyId": "6109efe96f903dc20f391c57b0ade122", "testCaseId": "6109efe96f903dc20f391c57b0ade122", "fullName": "tests.test.vwoLoginTests.test_vwo_login_pf.TestVWOLogin#test_vwo_login_positive", "labels": [{"name": "feature", "value": "Login Test"}, {"name": "epic", "value": "VWO App"}, {"name": "tag", "value": "qa"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests.test.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login_pf"}, {"name": "subSuite", "value": "TestVWOLogin"}, {"name": "host", "value": "Chaitra"}, {"name": "thread", "value": "26788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.vwoLoginTests.test_vwo_login_pf"}]}